librz/analysis/p/analysis_bf.c

```diff
@@ -7,35 +8,463 @@
 #include <rz_asm.h>
 #include <rz_analysis.h>
 
-static size_t countChar(const ut8 *buf, int len, char ch) {
```

Comment:
Seems unrelated to IL.

Reply:
It's an unused code. Rightfully removed.